src/ducks/apps/components/Application.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+import { translate } from 'cozy-ui/react/I18n'
+import Icon from 'cozy-ui/react/Icon'
+import Spinner from 'cozy-ui/react/Spinner'
+
+import UninstallModal from './UninstallModal'
+
+import defaultAppIcon from '../../../assets/icons/icon-cube.svg'
+
+export class ApplicationComponent extends Component {
+  constructor (props) {
+    super(props)
+    props.fetchInstalledApps()
```

Comment:
You shouldn't need to do this again here since it should be done by the parent, so the redux store should contains this information.